modules/ROOT/pages/manage-merges.adoc

```diff
@@ -25,12 +25,12 @@ The target type does not need to have the same name.
 +
 * (Recommended) Join fields from the two types to have a more enriched query result for this type.
 +
-If you make this selection, you are taken to the *Enable collaboration* pane, where you set a default query method and primary key. (Refer to  xref:collaboration.adoc[Enable Collaboration on a Type] for more information on object type collaboration.)
+If you make this selection, you are taken to the *Enable collaboration* pane, where you set a default query method and a primary key, or two primary keys if the query method uses composite keys. (Refer to  xref:collaboration.adoc[Enable Collaboration on a Type] for more information on object type collaboration.)
 
 * Declare the current type as a reference so that it returns only the fields and dataset of the type it's merged with.
 . In the *Merge with another type* pane, click *Preview merge* results.
 +
-In the following example, the preview shows the target OrderResponse object type in the unified schema with new fields added after the OrderSummary object type is merged with it:
+In the following example, the preview shows the target *OrderResponse* object type in the unified schema with new fields added after the *OrderSummary* object type is merged with it:
```

Comment:
I am not sure if everything after "the preview shows" is included in that preview or not, given that new fields are added after the object type is merged with "it," which I had to look back to figure out the referent.
I wonder if something like this is more clear?
```suggestion
In the following example, the preview shows the target *OrderResponse* object type in the unified schema, including the new fields added after the *OrderSummary* object type is merged with *OrderResponse*:
```